### PR TITLE
8 frequency tables

### PR DESCRIPTION
**Pull request by ldavies99**
Creating a draft pull request for CI and reviews.

### CODE REVIEW

**Comment by RuSnook** [comment]
General comment: Would be good to add versions for dependencies. I'm getting warnings for deprecation [e.g. dplyr::select() wants to use all_of()] as I'm using a later version.

**Review by RuSnook — COMMENTED** [review]
Looks really good. There are a few style things that I've commented on, some apply throughout but not a big deal so I only suggested once. A couple of other suggestions for the tests where I think you could simplify things.
But yeah, looks really good.

File: R/derive_vars.R [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  data$peer_review_score <- ifelse(data$prac_review %in% high_vals, 1, 0)
+  data$AQUA_book_score <- ifelse(data$prac_AQUA_book %in% high_vals, 1, 0)
+  data$doc_score <- ifelse(data$doc_readme %in% high_vals & data$doc_comments %in% high_vals, 1, 0)
+
+  data$basic_rap_score <- rowSums(data[,c("use_open_source_score",
+                                          "open_code_score",
+                                          "version_control_score",
+                                          "peer_review_score",
+                                          "AQUA_book_score",
+                                          "doc_score")])
```
[the remark this answers is not in the document]
Comment (ldavies99) [reply R/derive_vars.R:112]: The new cols are used in summarise_rap_comp().

File: R/derive_vars.R [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  data$cont_integreation_score <- ifelse(data$prac_automated_QA == "Yes", 1, 0)
+  data$dep_management_score <- ifelse(data$prac_dir_structure == "Yes", 1, 0)
+
+  data$advanced_rap_score <- rowSums(data[,c("function_score",
+                                             "unit_test_score",
+                                             "function_doc_score",
+                                             "package_score",
+                                             "code_style_score",
+                                             "cont_integreation_score",
+                                             "dep_management_score")])
```
[the remark this answers is not in the document]
Comment (ldavies99) [reply R/derive_vars.R:162]: The new columns are used in summarise_rap_comp().

**Comment by joshlynchONS** [comment]
Looks good to me. I like the refactoring and readability updates using tidyverse.

**Review by joshlynchONS — APPROVED** [review]
Looks good. I like the refactoring and readability changes using tidyverse.

**Pull request merged**